### PR TITLE
Performance improvement

### PR DESCRIPTION
**Pull request by igor-tkachev**

### CODE REVIEW

**Review by MaceWindu — APPROVED** [review]

**Review by viceroypenguin — APPROVED** [review]

File: Source/LinqToDB/Mapping/MappingSchema.cs [hunk 1]
```diff
@@ -1645,14 +1661,12 @@ public StringComparer ColumnNameComparer
 		/// <returns>Mapping descriptor.</returns>
 		public EntityDescriptor GetEntityDescriptor(Type type)
```
Comment (MaceWindu) [inline Source/LinqToDB/Mapping/MappingSchema.cs:1662]: this method called a lot and was creating a lot of closure objects. With this change it's back due to use of `this`

**Pull request merged**